RecoParticleFlow/PFProducer/src/PFEGammaFilters.cc

```diff
@@ -111,7 +153,17 @@ bool PFEGammaFilters::passElectronSelection(const reco::GsfElectron & electron,
 
   //  cout << " My OLD MVA " << pfcand.mva_e_pi() << " MyNEW MVA " << electron.mva() << endl;
   if(electron.mva_e_pi() > ele_noniso_mva_) {
-    passEleSelection = true; 
+    if (validHoverE) {
+        passEleSelection = true; 
+    } else {
+        bool EE = (std::abs(electron.eta()) > 1.485); // for prefer consistency with above than with E/gamma for now
+        if ((electron.full5x5_sigmaIetaIeta() < badHcal_full5x5_sigmaIetaIeta_[EE]) &&
+            (std::abs(1.0-electron.eSuperClusterOverP())/electron.ecalEnergy()  < badHcal_eInvPInv_[EE]) && 
+            (std::abs(electron.deltaEtaSuperClusterTrackAtVtx() - electron.superCluster()->eta() + electron.superCluster()->seed()->eta())  < badHcal_dEta_[EE]) && // looser in case of misalignment
```

Comment:
Thanks for using the correct dEtaIn (which is dEtaInSeed) but could we replace it with  std::abs(electron.deltaEtaSeedClusterTrackAtVtx()) 
its exactly the same but just reduces the chances of mistakes.

Reply:
ok. I used this formulation because it was what the e/&gamma; cut-based twiki was instructing to do. 
I can happily change it to the shorter form.